src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -239,6 +239,28 @@ static Library()
                     returnBehavior: ReturnBehavior.AlwaysEvaluateAndReturnResult,
                     targetFunction: Concatenate)
             },
+            {
+                BuiltinFunctionsCore.ColorValue,
```

Comment:
Copilot also needs ColorFade, pretty please! :)

Reply:
On it... ;)